From 8ae00d377605a2c9dc4f541bab09ba7859b90e3c Mon Sep 17 00:00:00 2001
From: "Cui, Yuxin" <yuxin.cui@intel.com>
Date: Mon, 18 Jul 2022 09:55:52 +0800
Subject: [PATCH 1/2] Change the path of ueventd.rc

Update the path of ueventd.rc because Android T doesn't support legacy
ueventd configuration file.

Tracked-On: OAM-102903
Signed-off-by: Cui, Yuxin <yuxin.cui@intel.com>
---
 groups/device-specific/caas_dev/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/device-specific/caas_dev/product.mk b/groups/device-specific/caas_dev/product.mk
index 4a104c0..6a6531f 100755
--- a/groups/device-specific/caas_dev/product.mk
+++ b/groups/device-specific/caas_dev/product.mk
@@ -8,7 +8,7 @@ PRODUCT_COPY_FILES += \
     $(LOCAL_PATH)/init.recovery.rc:root/init.recovery.$(TARGET_PRODUCT).rc \
 {{#treble}}
     $(LOCAL_PATH)/init.rc:$(TARGET_COPY_OUT_VENDOR)/etc/init/hw/init.$(TARGET_PRODUCT).rc \
-    $(LOCAL_PATH)/ueventd.rc:$(TARGET_COPY_OUT_VENDOR)/ueventd.rc \
+    $(LOCAL_PATH)/ueventd.rc:$(TARGET_COPY_OUT_VENDOR)/etc/ueventd.rc \
 {{/treble}}
 {{^treble}}
     $(LOCAL_PATH)/init.rc:root/init.$(TARGET_PRODUCT).rc \
-- 
2.17.1

